themes/sandal/less/common.less

```diff
@@ -12,12 +12,11 @@
 
 .highlight,
 mark { background-color: @highlighter-color; }
-table { font-size: @reduced-font-size; }
 
 /* ------ Bootstrap class overrides ------ */
 .badge {
   margin-bottom: 1px;
-  font-size: 10px;
+  font-size: ceil((@reduced-font-size * .65)); // ~10px
```

Comment:
Should we make a @small-font-size to standardize this across files? Making it a static 10px instead of typing it to might be the best option for accessibility. We could even call it @min-font-size.

Reply:
Done.